### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by YAHYAAMRANI** [johngodley/redirection#2842]
```
Plugin: 5.0.1
WordPress: 5.7 (single)
PHP: 7.4.12
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36
JavaScript: https://gifsviralme-82b4eb.ingress-comporellon.easywp.com/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3
REST API: https://gifsviralme-82b4eb.ingress-comporellon.easywp.com/wp-json/
Query: ?page=redirection.php&orderby=source
TypeError: Cannot read property 'url' of null

    at a (https://gifsviralme-82b4eb.ingress-comporellon.easywp.com/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:666171)
    at f (https://gifsviralme-82b4eb.ingress-comporellon.easywp.com/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:373379)
    at td
    at Nh (https://gifsviralme-82b4eb.ingress-comporellon.easywp.com/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:602326)
    at tr
    at Dh (https://gifsviralme-82b4eb.ingress-comporellon.easywp.com/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:602820)
    at tbody
    at table
    at Uh (https://gifsviralme-82b4eb.ingress-comporellon.easywp.com/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:603773)
    at jm (https://gifsviralme-82b4eb.ingress-comporellon.easywp.com/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:619391)
    at div
    at https://gifsviralme-82b4eb.ingress-comporellon.easywp.com/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:714082
    at f (https://gifsviralme-82b4eb.ingress-comporellon.easywp.com/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:373379)
    at fw (https://gifsviralme-82b4eb.ingress-comporellon.easywp.com/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:716693)
    at zf (https://gifsviralme-82b4eb.ingress-comporellon.easywp.com/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:532810)
    at div
    at a (https://gifsviralme-82b4eb.ingress-comporellon.easywp.com/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:477523)
    at https://gifsviralme-82b4eb.ingress-comporellon.easywp.com/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:719894
    at f (https://gifsviralme-82b4eb.ingress-comporellon.easywp.com/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:373379)
    at We (https://gifsviralme-82b4eb.ingress-comporellon.easywp.com/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:370986)
    at Cw
```

**Comment by JD-aus** [comment]
Similar error here:

Plugin: 5.0.1
WordPress: 5.7 (single)
PHP: 7.3.27
Browser: Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.114 Safari/537.36
JavaScript: https://www.securetech.com.au/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3
REST API: https://www.securetech.com.au/wp-json/
Query: ?page=redirection.php&orderby=last_count
TypeError: Cannot read property 'url' of null

    at a (https://www.securetech.com.au/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:666171)
    at f (https://www.securetech.com.au/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:373379)
    at td
    at Nh (https://www.securetech.com.au/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:602326)
    at tr
    at Dh (https://www.securetech.com.au/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:602820)
    at tbody
    at table
    at Uh (https://www.securetech.com.au/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:603773)
    at jm (https://www.securetech.com.au/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:619391)
    at div
    at https://www.securetech.com.au/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:714082
    at f (https://www.securetech.com.au/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:373379)
    at fw (https://www.securetech.com.au/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:716693)
    at zf (https://www.securetech.com.au/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:532810)
    at div
    at a (https://www.securetech.com.au/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:477523)
    at https://www.securetech.com.au/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:719894
    at f (https://www.securetech.com.au/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:373379)
    at We (https://www.securetech.com.au/wp-content/plugins/redirection/redirection.js?ver=5.0.1-3aaca2e8b96d0af2cc585f8590d6feb3:2:370986)

**Comment by johngodley** [comment]
Is this fixed in 5.1.1?

**Comment by johngodley** [comment]
Closing for now, please reopen with more information.
